plugins/commands/autocomplete/command/install.rb

```diff
@@ -0,0 +1,34 @@
+require 'optparse'
+
+require 'vagrant/util/install_cli_autocomplete'
+
+module VagrantPlugins
+  module CommandAutocomplete
+    module Command
+      class Install < Vagrant.plugin("2", :command)
+        def execute
+          options = {}
+
+          opts = OptionParser.new do |o|
+            o.banner = "Usage: vagrant autocomplete install [-h]"
```

Comment:
Is it possible for us to have flags based on which shell to install? Not sure how hard that would be to add.

Reply:
I think it's possible. What do you envision being different between shells?
I was thinking options would be like: `--append` where it appends the blrub, `--prepend` where it prepends the blurb, `--shell` that allows you to choose which shell you want to install.

Reply:
I'm mostly thinking people might want an option to only install one of the available shells, instead of every possible one. So maybe it'd be like `autocomplete install --bash` for example.

Reply:
Ya, I like that. Should users be allowed to specify multiple shells like: `autocomplete install --bash --zsh`
Then the output of `autocomplete install [-h]` looks something like 
```
Usage: vagrant autocomplete install [-h]

Options:

    -b, --bash                       Install bash autocomplete
    -z, --zsh                        Install zsh autocomplete
        --[no-]color                 Enable or disable color output
        --machine-readable           Enable machine readable output
    -v, --version                    Display Vagrant version
        --debug                      Enable debug output
        --timestamp                  Enable timestamps on log output
        --debug-timestamp            Enable debug output with timestamps
        --no-tty                     Enable non-interactive output
    -h, --help                       Print this help
```

Another option in the same line of thinking is something like `autocomplete install <shell>`
Then the output of `autocomplete install [-h]` looks something like :
```
Usage: vagrant autocomplete install [-h] [shell name]

Available shells: zsh bash

Options:

        --[no-]color                 Enable or disable color output
        --machine-readable           Enable machine readable output
    -v, --version                    Display Vagrant version
        --debug                      Enable debug output
        --timestamp                  Enable timestamps on log output
        --debug-timestamp            Enable debug output with timestamps
        --no-tty                     Enable non-interactive output
    -h, --help                       Print this help
```
I like (2) since it doesn't require developers to change the cli when adding more shells, and it more closely resembles other commands in vagrant like `box add <name>` Though `vagrant autocomplete install zsh` does seem pretty clunky. Any thoughts on this?

Reply:
Yeah! I think it makes sense to keep it a flag, unless each shell could have additional configuration options? :thinking: Otherwise I think what you have on that first output with the shells being flags looks good to me! :+1: